diagnostic.md

```diff
@@ -27,7 +27,9 @@ on the body element. Use that variable to increase font-size of `h1`
 elements by 30%.
 
 ```scss
-// your response here
+h1 {
+  font-size: ($base-font-size * 30%)
```

Comment:
Correct usage but we want the `h1`'s to be **increased** by 30%. So it would need to be `font-size: $base-font-size * 1.3;`